approval_polls/views.py

```diff
@@ -36,6 +37,12 @@ class DetailView(generic.DetailView):
   def get_queryset(self):
       return Poll.objects.filter(pub_date__lte=timezone.now())
 
+@login_required
+def delete_poll(request):
```

Comment:
I meant to rewrite the view named `DetailView` that inherits from `generic.DetailView` so that it inherits from [`generic.View`](https://docs.djangoproject.com/en/1.8/ref/class-based-views/base/#view) and has methods `get` (that does what `DetailView` was doing earlier) and `delete` (that performs poll deletion from the db). I pointed to the CreateView class because it does something similar.